compiler/aarch64/codegen/OMRMemoryReference.cpp

```diff
@@ -438,30 +438,18 @@ void OMR::ARM64::MemoryReference::populateMemoryReference(TR::Node *subTree, TR:
       }
    else
       {
-      if (subTree->getOpCodeValue() == TR::aiadd ||
-          subTree->getOpCodeValue() == TR::iadd)
```

Comment:
These lines were brought from OMRMemoryReference for arm32.
The address calculation on AArch64 is always 64-bit, and these conditions have never been met.